File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.js
function solution(s) {

    let eWords = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine'];
   
    for(var i=0; i < eWords.length; i++) {
        s = s.replace(new RegExp(eWords[i], 'g'), i);
    }
    let answer = Number(s);
    return answer;
}

File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.java
class Solution {
	public int solution(String s) {
		
		String [] eWords = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"}; 
		
		for(int i=0; i < eWords.length; i++) {
			s = s.replace(eWords[i], Integer.toString(i));
			
		}
		int answer = Integer.parseInt(s);
		
		return answer;				
	}
	
}